src/components/touchables/GenericTouchable.tsx

```diff
@@ -14,61 +24,51 @@ import PropTypes from 'prop-types';
  * travel outside it transits to special MOVED_OUTSIDE state. Gesture recognition
  * finishes in UNDETERMINED state.
  */
-export const TOUCHABLE_STATE = {
+export const TOUCHABLE_STATE: Record<string, TouchableStateValuesType> = {
   UNDETERMINED: 0,
   BEGAN: 1,
   MOVED_OUTSIDE: 2,
 };
 
-const PublicPropTypes = {
+type TouchableStateValuesType = 0 | 1 | 2;
+
+export interface GenericTouchableProps extends TouchableWithoutFeedbackProps {
   // Decided to drop not used fields from RN's implementation.
-  // e.g. onBlur and onFocus as well as deprecated props.
-  accessible: PropTypes.bool,
-  accessibilityLabel: PropTypes.node,
-  accessibilityHint: PropTypes.string,
-  hitSlop: PropTypes.shape({
-    top: PropTypes.number,
-    left: PropTypes.number,
-    bottom: PropTypes.number,
-    right: PropTypes.number,
-  }),
-  disabled: PropTypes.bool,
-  onPress: PropTypes.func,
-  onPressIn: PropTypes.func,
-  onPressOut: PropTypes.func,
-  onLayout: PropTypes.func,
-  onLongPress: PropTypes.func,
-  nativeID: PropTypes.string,
-  testID: PropTypes.string,
-  delayPressIn: PropTypes.number,
-  delayPressOut: PropTypes.number,
-  delayLongPress: PropTypes.number,
-  shouldActivateOnStart: PropTypes.bool,
-  disallowInterruption: PropTypes.bool,
-};
+  // e.g. onBlur and onFocus as well as deprecated props. - TODO: this comment may be unuseful in this moment
```

Comment:
I am not sure what should we do about it.

Reply:
If it seems not useful for you maybe we should delete it? If someone will spot there's something missing, git history for that file is up for exploration. For others, I think it might be just noise. Let me know what you think